src/MICore/UnixUtilities.cs

```diff
@@ -55,7 +61,7 @@ public static class UnixUtilities
                 // we need to fake an exit by the debugger
                 "pid=$! ; " +
                 "echo $pid > {3}; " +
-                "wait $pid; ",
+                waitForCompletionCommand,
```

Comment:
nit: You might want to consider having this as an argument to `string.Format`. I don't have a strong opinion about this, but it might make it clearer since all the lines below are actual arguments to `string.Format`.